content/docs/instrumenting/standard_exports.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Standard Exports
+sort_rank: 7
+---
+
+# Standard Exports
+
+This document covers what exports and collectors Prometheus client libraries
+should offer, with the aim of consistency across libraries, making the easy use
+cases easy and making it easier to provide tooling and consoles for common use
+cases.
+
+In addition, client libraries are ENCOURAGED to also offer whatever makes sense
+in terms of metrics for their language’s runtime (e.g. Garbage collection
+stats).
+
+These exports should have the prefix `process_`. If a language or runtime
```

Comment:
The current wording above make it sound like standard exports include runtime metrics. It might make sense to improve the wording of the definition then.